apps/token/src/components/nav/nav-dropdown.tsx

```diff
@@ -0,0 +1,206 @@
+import classNames from 'classnames';
```

Comment:
Question (non-blocking): I know a lot of the navbar work is now shareable between apps, shouldn't this be in the ui-toolkit too?